### DISCUSSION TITLE
DCDAO Budget Adjustment Recommendations and Reminder

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#183] [General]
#### 1. Budget Mode Background and Explanation
   - **Background**: With increasing market volatility, DCDAO needs a more flexible budget model to cope with changing conditions. If the bear market persists, the required amount of RING may exceed the Darwinia Treasury's budget limit, making it difficult to maintain sufficient operating funds (decentralized projects typically require a runway of 3 years or more to ensure sustained ecosystem development in uncertain market conditions). Therefore, I recommend that DCDAO switch between bear or bull market budget modes to better manage spending and ensure long-term sustainability.
   
   - **Bear Market Mode**: Monthly cap of $14,000 and 9,000,000 RING. (Current Mode)
   - **Bull Market Mode**: Monthly cap of $20,000 and 4,000,000 RING.

#### 2. Budget Control Measures
   - **Steering Meeting**: Already adjusted to bi-weekly, we recommend reducing the subsidy from $1,000 to $500 accordingly.
   - **Community Program**: Adjusted to $4,000.
   - **Treasury Specialist**: Adjusted to $600.
   - **Community Growth Program**: Adjusted to $1,500.
   - **Marketing and Branding Program**: Infura Program has been canceled (already approved and implemented), and other budgets including frontend can adjusted to $3,500.

#### 3. Overall Recommendations
   - **Flexible Mode Switching**: Switch between bear and bull market modes based on market changes to ensure the budget aligns with market conditions.
   - **Regular Evaluation**: Review budget performance monthly or quarterly to ensure efficient resource allocation and make adjustments as needed to avoid shortfalls.
   - **Transparent Management**: Strengthen budget transparency through reporting and timely communication, and regularly update DAO members on expenditures to ensure community engagement and governance.

#### 4. Reminder
   - Q3 has ended, and existing programs are updated quarterly and do not auto-renew. As we enter Q4, all programs, if they desire to continue, should be updated, submitted, and voted on for approval before they can take effect.

This budget adjustment plan helps optimize spending during bear markets, ensuring core project operations, while allowing for increased investment during bull markets to foster long-term growth.

**Comment by SasoLithops** [comment]
I agree with bear market mode. Not good to drain treasury. If in  bull market we do many x I recommend to treasury specialist to move some of Ring to stables and BTC. To be better prepared for next bear.
